JSON_files/weapons.JSON

```diff
@@ -0,0 +1,2249 @@
+{
+  "Choppa": {
+    "abbreviation": "Chop",
+    "attack": {
+      "standard": 28,
+      "master_crafted": 32,
+      "artifact": 36
+    },
+    // ... (other attributes)
+  },
+  "Snazzgun": {
+    "abbreviation": "Snazz",
+    "attack": {
+      "standard": 80,
+      "master_crafted": 92,
+      "artifact": 104
+    },
+    // ... (other attributes)
+  },
+  "Shuriken Pistol": {
+    "abbreviation": "ShurikP",
+    "attack": {
+      "standard": 25,
+      "master_crafted": 28,
+      "artifact": 31
+    },
+    // ... (other attributes)
+  },
+  "Storm Shield": {
+    "abbreviation": "StShield",
+    "attack": {
+      "standard": 0,
+      "master_crafted": 0,
+      "artifact": 0
+    },
+    // ... (other attributes)
+  },
+  "Boarding Shield": {
+    "abbreviation": "BoShield",
+    "attack": {
+      "standard": 0,
+      "master_crafted": 0,
+      "artifact": 0
+    },
+    // ... (other attributes)
+  },
+  "Hellgun": {
+    "abbreviation": "HGun",
+    "attack": {
+      "standard": 30,
+      "master_crafted": 34,
+      "artifact": 38
+    },
+    // ... (other attributes)
+  },
+  "Hellrifle": {
+    "abbreviation": "HRifle",
+    "attack": {
+      "standard": 150,
+      "master_crafted": 160,
+      "artifact": 170
+    },
+    // ... (other attributes)
+  },
+"Archeotech Laspistol": {
+    "attack": {
+        "standard": 120,
+        "master_crafted": 130,
+        "artifact": 140
+    },
+    "melee_mod": {
+        "standard": 0,
+        "master_crafted": 0,
+        "artifact": 0
+    },
+    "ranged_mod": {
+        "standard": 0,
+        "master_crafted": 0,
+        "artifact": 0
+    },
+    "hp_mod": {
+        "standard": 0,
+        "master_crafted": 0,
+        "artifact": 0
+    },
+    "damage_resistance_mod": {
+        "standard": 0,
+        "master_crafted": 0,
+        "artifact": 0
+    },
+    "description": "Known as a Lasrod or Gelt Gun, this pistol is an ancient design of Laspistol with much greater range and power.",
+    "melee_hands": 0,
+    "ranged_hands": 1,
+    "ammo": 0,
+    "range": 3.1,
+    "spli": 0,
+    "arp": 0
+},
+"Combat Knife": {
+    "attack": {
+        "standard": 25,
+        "master_crafted": 30,
+        "artifact": 35
+    },
+    "melee_mod": {
+        "standard": 0,
+        "master_crafted": 0,
+        "artifact": 0
+    },
+    "ranged_mod": {
+        "standard": 0,
+        "master_crafted": 0,
+        "artifact": 0
+    },
+    "hp_mod": {
+        "standard": 0,
+        "master_crafted": 0,
+        "artifact": 0
+    },
+    "damage_resistance_mod": {
+        "standard": 0,
+        "master_crafted": 0,
+        "artifact": 0
+    },
+    "description": "More of a sword than a knife proper, this tough and thick blade becomes a deadly weapon in the hand of an Astartes.",
+    "melee_hands": 0,
+    "ranged_hands": 0,
+    "ammo": 0,
```

Comment:
would be simpler to skip defining props with 0 values and simply set default 0 for these during load